ga/GameAnalytics.gd

```diff
@@ -0,0 +1,539 @@
+class_name GameAnalytics
+extends Node
+# GameAnalytics <https://gameanalytics.com/> GDScript REST API implementation
+
+signal init(status_code)
+
+# Platform remaps
+const PLATFORMS = {
+	'Windows': 'windows',
+	'X11': 'linux',
+	'OSX': 'mac_osx',
+	'Android': 'android',
+	'iOS': 'ios',
+	'HTML5': 'webgl',
+}
+
+# Available API endpoints
+const INIT_ENDPOINT = '/v2/%s/init'
+const EVENTS_ENDPOINT = '/v2/%s/events'
+
+# NOTE: This plugin is using the "rest api v2" but GA docs
+# supports "godot [0-9]{0,5}(\\.[0-9]{0,5}){0,2})"
```

Comment:
Although Godot is a supported value for `sdk_version`, the docs also say that

> Custom solutions should ALWAYS use the string “rest api v2”

Maybe sending "godot" from what is a "Custom solution" messes with their own metrics about the standard solution and shouldn't be done. I'm speculating, of course, but it might be a good idea to check with GA first.

Reply:
You're right. I did it that way because at the time there was no official implementation.